metaflow/metadata/metadata.py

```diff
@@ -497,41 +496,77 @@ def _metadata_to_json(self, run_id, step_name, task_id, metadata):
             for datum in metadata
         ]
 
-    def _tags(self):
+    def _get_system_info_as_dict(self):
+        """This function drives:
+        - sticky system tags initialization
+        - task-level metadata generation
+        """
+        sys_info = dict()
         env = self._environment.get_environment_info()
-        tags = [
-            resolve_identity(),
-            "runtime:" + env["runtime"],
-            "python_version:" + env["python_version_code"],
-            "date:" + datetime.utcnow().strftime("%Y-%m-%d"),
-        ]
+        sys_info["runtime"] = env["runtime"]
+        sys_info["python_version"] = env["python_version_code"]
+        sys_info["date"] = datetime.utcnow().strftime("%Y-%m-%d")
```

Comment:
see https://github.com/Netflix/metaflow/pull/917#discussion_r791079881

Reply:
Removed, updated PR summary with:
```
Drive-by: we also remove date as a piece of system information. I.e. it will no longer show up as a system tag for ALL objects, It is too-generically named and similar information is available in client API as created_at.
```